apps/mobile/src/screens/wheel/wheel.tsx

```diff
@@ -33,7 +33,7 @@ const Wheel: React.FC = () => {
 		navigation.navigate(BottomTabScreenName.EDIT_WHEEL_RESULTS);
 	}, [navigation]);
 
-	const date = getFormattedDate("d MMM yyyy, EEEE");
+	const date = getFormattedDate(new Date().toDateString());
```

Comment:
I changed the argument in mobile as I updated shared getFormattedDate helper.
In mobile the date was just the current day. That bug is described by QA's under one of mobile tickets